content/docs/pipelines/output-viewer.md

```diff
@@ -145,3 +145,16 @@ must be self-contained, with no references to other files in the filesystem. It
 can still have absolute references to files on the web, however. Content running 
 inside this web app is isolated in an iframe, and cannot communicate with the 
 Kubeflow Pipelines UI.
+
+## Markdown
+
+**type:** `'markdown'`
+
+**Metadata fields:**
+
+- `storage`
+- `source`
+
+Renders markdown strings in the output. The markdown data can either be read from
+a file stored remotely, or can be embedded in the metadata field `source`, in
+which case the `storage` field's value must be `"inline"`.
```

Comment:
For consistency with the rest of the doc, use single quotation marks: `'inline'`